crud/ClassCRUD/src/ClassCRUD.java

```diff
@@ -0,0 +1,70 @@
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+public class ClassCRUD {
+
+	public void insertClass(Statement st, String class_id, String stream_id, String user_id){
+		try {
+			System.out.println("Inserting a class...");
+			int rows = st.executeUpdate("INSERT INTO Student_Performance.Class VALUES ('" + class_id + "','" + stream_id + "','" + user_id  + "')");
```

Comment:
https://xkcd.com/327/

SQL Injection is serious. Should at the very least use a prepared statement. Can still be modular that way, but not vulnerable. Prepared statement can be written entirely inside Java.

Reply:
Use prepared statement or simmilar: https://www.javacodegeeks.com/2012/11/sql-injection-in-java-application.html

Seriously, use prepared statement WITH `?` PARAMATERS https://vladmihalcea.com/a-beginners-guide-to-sql-injection-and-how-you-should-prevent-it/

Reply:
I notice it says Oracle will reject it.... but it is still terrible practice, and if you do that on MySQL you are terrible vulnerable.